### ISSUE TITLE
Icon Request: icon-password-textbox

### ISSUE THREAD

**Issue by omkartapale** [FortAwesome/Font-Awesome#11156]
FA don't have any icon representing password. I need icon to use for password input
Example of what I'm looking for:
https://materialdesignicons.com/icon/textbox-password

**Comment by schwartzadev** [comment]
Have you tried [lock](http://fontawesome.io/icon/lock/) or [key](http://fontawesome.io/icon/key/)?

**Comment by omkartapale** [comment]
@schwartzadev Yes, tried both. But looking for more appropriate for encrypted text

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
